.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please add `.vscode` to your `.gitignore` file in home directory.

Reply:
Yes I did that and the new PR doesn't include change of .gitignore